test/routes/admin/licenceSearch.test.js

```diff
@@ -71,12 +64,5 @@ describe('/licenceSearch/', () => {
     })
   })
 
-  function createApp(user) {
-    const prisonerService = createPrisonerServiceStub()
-    const licenceService = createLicenceServiceStub()
-    const signInService = createSignInServiceStub()
-    const baseRouter = standardRouter({ licenceService, prisonerService, audit: auditStub, signInService })
-    const route = baseRouter(createAdminRoute(licenceSearchService))
-    return appSetup(route, user, '/admin/licenceSearch')
-  }
+  const createApp = (user) => startRoute(createAdminRoute(licenceSearchService), '/admin/licenceSearch', user)
```

Comment:
much nicer way of creating test apps for web testing. 

It does't quite work for tests that rely on the middleware automatically loading licences which will require a bit of work.